documentation/interrupt_handling.md

```diff
@@ -55,23 +109,35 @@ Some general purpose registers are not saved and restored on entry and
 exit, those are used to pass parameters between secure and normal world
 (see ARM_DEN0028A_SMC_Calling_Convention for details).
 
-# Entry and exit of Trusted OS
+## 2.2. Entry and exit of Trusted OS
 On entry and exit of Trusted OS each CPU is uses a separate entry stack and
 runs with IRQ and FIQ blocked.
 
 During the entry phase a context is selected to start/resume execution in.
+Inside optee_os, the context is managed by a so-called TEE thread.
```

Comment:
As far as I know we're not calling it TEE threads anywhere, "trusted OS thread" perhaps?

Reply:
optee_os thread would fit ?

Reply:
@igoropaniuk : here we should notify that optee_os threads are assigned only to standard services (i mean not fast SMCs).